### PR TITLE
New design

### PR DESCRIPTION
**Pull request by thefoxis**
**Overview**
- removes unnecessary images and SVG files
- introduces new, more readable styling
- cleans up Jade/HTML
- adds throwing errors on compile, thanks to #3
- updates `.gitignore`
- updates README.md with `human-model` and `human-view` deps

**Before merging into** `gh-pages`
- Fix the `slider` positioning in the navigation
- Add `active` class to the link corresponding to slider position
- Clean up headings in `human-view` and `human-model` for proper structure and hierarchy in the navigation

:gift_heart:

### CODE REVIEW

**Comment by ghost** [comment]
Hello @thefoxis. The design is quite lovely.
Just a heads-up -- when I ran it on my local, the console complained: human-javascript.png was missing from static/images. However, humanjs.svg exists in that directory. After swapping the png for the svg, everything is fine:

``` jade
//- index.jade:136, 
img(src="static/images/humanjs.svg", width="200", alt="human javascript ebook")
```

``` html
<!-- index.html:118, after $ npm build -->
<p> <a href="http://humanjavascript.com" class="image-wrapper humanjs"><img src="static/images/humanjs.svg" width="200" alt="human javascript ebook"></a></p>@gendoc human-model ../human-model/README.md
```

Is human-javascript.png missing, or does humanjs.svg take its place?

**Comment by thefoxis** [comment]
ah, shoot! thanks for catching this @mizuhands :) I accidentaly removed the file when I was getting rid of other unused images. works now.

**Comment by ghost** [comment]
You are welcome! 
Also possible typo -- moonbots, or moonboots?

``` jade
//- index.jade:152,153
h2(data-nav="moonbots")
  a(href="https://github.com/henrikjoreteg/moonboots") Moonbots
```

**Comment by thefoxis** [comment]
Nothing better than a thorough review, especially when I'm a typo master. Thanks :sunflower:

**Comment by ghost** [comment]
Again, you are welcome. Glad I could help :octocat: !

**Comment by HenrikJoreteg** [comment]
@thefoxis could you style these as well: https://cloudup.com/ccbOZq1kSf5 you'll notice some of the args in the API docs are actually wrapped in a `<span class="type">` it'd be cool if they had a bit more spacing or were a different color or something so they don't just blend into the sentence that follows it.

File: static/css/main.styl [hunk 1]
```diff
@@ -1,439 +1,10 @@
...
-      background: brightblue
-      roundall(3px)
-      padding: 0 4px 
-
-  > a:last-child
-    margin-bottom: 100px
-
-  #slider
-    position: absolute
-    transition(all, 250ms, ease-out, 0);
```
Comment (HenrikJoreteg) [inline static/css/main.styl:223]: I kinda liked having this animate up and down as you scroll.

**Comment by HenrikJoreteg** [comment]
We lost whitespace in code sections somehow, so they're just one line: https://cloudup.com/cIt8Frw8tlP

**Comment by thefoxis** [comment]
@HenrikJoreteg &mdash; white space is fixed, transition added back, inline code has more prominent background (hope that helps, looking at other styling examples I don't think we should make it stand out more, good enough in my opinion) and fixed a bug with negative margins.

Hope I didn't forget anything :)

**Pull request merged**